yjit/src/codegen.rs

```diff
@@ -1771,40 +1774,39 @@ fn gen_checkkeyword(
     KeepCompiling
 }
 
-/*
 fn gen_jnz_to_target0(
-    cb: &mut CodeBlock,
+    asm: &mut Assembler,
     target0: CodePtr,
     _target1: Option<CodePtr>,
     shape: BranchShape,
 ) {
     match shape {
         BranchShape::Next0 | BranchShape::Next1 => unreachable!(),
-        BranchShape::Default => jnz_ptr(cb, target0),
+        BranchShape::Default => asm.jnz(Target::CodePtr(target0)),
```

Comment:
`target0.into()`

Reply:
addressed in https://github.com/Shopify/ruby/pull/352/commits/93f02842ea74691481607812e21c5da1fdcdf516